libethereum/State.h

```diff
@@ -142,8 +141,7 @@ struct Change
 	{}
 };
 
-}
-
+} //detail
```

Comment:
Ugly formatting. Put 2 spaces before `//`, and space after `//`. Also better comment: `// namespace detail`.